src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/integration/mockservers/PrisonRegisterIntegrationTest.kt

```diff
@@ -96,4 +93,24 @@ class PrisonRegisterIntegrationTest(
 
     assertThat(result, absent())
   }
+
+  @Test
+  fun `handle timeout from client`() {
```

Comment:
Still wondering about these tests going to another test class with a parameterised test across all the classes which now have this behaviour.  Maybe on a future PR.